### PR TITLE
Docsify implementation

### PR DESCRIPTION
**Pull request by apeart-rd**
Implemented Docsify and transferred all existing confluence documentation under Docs (draft).

### CODE REVIEW

**Review by taslangraham — CHANGES_REQUESTED** [review]
- Some navigations do not work because of incorrect casing on files referenced inside _sidebar.md
- We need a README.md file that loads when we are at '/' 
- Adding a cover/landing page would be cool - https://docsify.js.org/#/cover

**Comment by OrandiH** [comment]
- Not sure where this is coming from on the main screen - https://share.getcloudapp.com/bLu0qL6Z
- I do think we need a theme.
- Same comment as Taslan re navigations.

File: docs/GettingStarted.md [hunk 1]
```diff
@@ -6,62 +6,69 @@
...
 
-### Step 3: Serve project
+## About
 
-```
-npm run serve
-```
+The RDVue CLI is the product of RealDecoy's Frontend Practice group. Contributions are welcome! You can help us by reporting or fixing bugs and giving us feedback on new/existing features.
 
 The project will be served at [http://localhost:8080/](http://localhost:8080/) by default. This information will also be printed out in your terminal. Visiting the link the app is served on will display a default page which was created on project creation.
```
Comment (taslangraham) [inline docs/GettingStarted.md:73]: The entire getting started documentation is replaced here. Is this intentional?

File: docs/index.html [hunk 2]
```diff
@@ -19,7 +19,7 @@
       homepage: 'gettingStarted.md',
```
Comment (taslangraham) [inline docs/index.html:19]: The getting started document has incorrect info in it.  Update it with the contents on confluence

File: docs/index.html [hunk 3]
```diff
@@ -19,7 +19,7 @@
...
       loadSidebar: '_sidebar.md',
       name: 'RDVue Documentation',
-      repo: 'https://github.com/realdecoy/rdvue/tree/feature/rdvue-2.0',
+      repo: 'https://github.com/realdecoy/rdvue/',
```
Comment (taslangraham) [inline docs/index.html:22]: Hey, could we add a cover page as well? https://docsify.js.org/#/cover

**Review by taslangraham — CHANGES_REQUESTED** [review]
Please see comments made throughout.

Not sure why, but only sub-sections has highlighted navigations. See images:
1)https://share.getcloudapp.com/p9urNy2r - sub-section
2)https://share.getcloudapp.com/rRu0WKxv - a main section. Navigating to it doe not give the blue highlight like in the previous picture

File: docs/_sidebar.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+  - [**Getting Started**](/)
```
Comment (OrandiH) [inline docs/_sidebar.md:1]: I noticed there was a issue with the layout for the items for CLI Commands, please see link here -> https://share.getcloudapp.com/2Nu0vwA8

**Review by OrandiH — CHANGES_REQUESTED** [review]
Please see comments.

**Review by OrandiH — APPROVED** [review]

**Review by taslangraham — APPROVED** [review]

**Pull request merged**